app/volume.go

```diff
@@ -31,6 +33,10 @@ const (
 	SnapshotPurgeStatusInterval = 5 * time.Second
 
 	WaitInterval = 5 * time.Second
+	timeout      = 300 // 5 minutes
```

Comment:
Give `timeout` a name, e.g. `VolumeAttachTimeout`.

Reply:
Thank you. Fixed